helpers/file/__init__.py

```diff
@@ -19,6 +18,13 @@ def read_csv_dict_file(file_path):
 			for row in reader:
 				yield row
 
-
 def is_empty(path):
 	return os.stat(path).st_size==0
+
+def removeDirectory(dirPath):
+	''' remove directory recursively '''
```

Comment:
could you please use variable name as dir_path and method name as remove_directory